typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumBscLumiaLUMIAWarpConfig.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  ChainMap,
+  RouterConfig,
+  TokenRouterConfig,
+  TokenType,
+} from '@hyperlane-xyz/sdk';
+import { objMap } from '@hyperlane-xyz/utils';
+
+// Lumia Team
+const owner = '0x8bBA07Ddc72455b55530C17e6f6223EF6E156863';
+
+export const getEthereumBscLUMIAWarpConfig = async (
+  routerConfig: ChainMap<RouterConfig>,
+): Promise<ChainMap<TokenRouterConfig>> => {
+  const ethereum = {
+    type: TokenType.collateral,
+    token: '0xD9343a049D5DBd89CD19DC6BcA8c48fB3a0a42a7',
```

Comment:
I would define the token address here https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/d8a9ad7abf6f66557d916dad90068d985be52460/typescript/infra/src/config/warp.ts#L5

Reply:
I dont expect this to be reused